BoxContentSDK/BoxContentSDKTests/BOXFileThumbnailRequestTests.m

```diff
@@ -98,6 +98,15 @@ - (void)test_that_thumbnail_request_returns_expected_thumbnail_after_intermediat
     XCTAssertEqual(1 + numberOfIntermediate202Responses, numberOfOperationsEnqueued);
 }
 
+- (void)test_that_operation_is_marked_as_thumbnail_download
+{
+    BOXFileThumbnailRequest *request = [[BOXFileThumbnailRequest alloc] initWithFileID:@"123" size:BOXThumbnailSize64];
+    BOXAPIOperation *operation = [request operation];
+    XCTAssert([operation isKindOfClass:[BOXAPIDataOperation class]]);
+    BOXAPIDataOperation *dataOperation = (BOXAPIDataOperation *)dataOperation;
+    XCTAssert(dataOperation.isSmallDownloadOperation);
```

Comment:
Could you please also add a test for regular downloads that asserts that this is false. Thanks!